app/templates/briefs/not_is_supplier_eligible_for_brief_error.html

```diff
@@ -1,43 +1,27 @@
 {% extends "_base_page.html" %}
 
-{% block page_title %}Not eligible for brief – Digital Marketplace{% endblock %}
+{% block page_title %}Not eligible for opportunity – Digital Marketplace{% endblock %}
 
 {% block main_content %}
 
-{% if clarification_question %}
+{% with inhibited_action="ask a question about" if clarification_question else "apply for" %}
```

Comment:
Quite possible, though done this way allows all the _language_ decisions remain together in the template.

What do people think? I'm easy.

@tombye you're a frontendy kinda guy...